gitops/charts/traffic-court-online/charts/maintenance-site/templates/route.yaml

```diff
@@ -0,0 +1,22 @@
+kind: Route
```

Comment:
add a `{{- if .Values.route.create }}` and a value that defaults to `true`

Reply:
Made it according to suggestions.